internal/translator/data/languages_test.go

```diff
@@ -59,4 +62,44 @@ func TestLanguages(t *testing.T) {
 			t.Fatalf("An error was excpected. Language does not exist")
 		}
 	})
+
+	t.Run("update a document when exists", func(t *testing.T) {
```

Comment:
My unique concern here is the structure of the name as I mentioned in the last PR. But it is not a blocker.